.kitchen.yml

```diff
@@ -20,14 +20,20 @@ platforms:
     image: centos-7-0-x64
 
 - name: ubuntu-12.04
+  run_list:
+    - recipe[apt]
```

Comment:
@patcon 
May be, it is better to move DO-specific settings to `.kitchen.digitalocean.yml`?
For example, like Chef does it for Docker driver: 
https://github.com/chef-cookbooks/database/blob/master/.kitchen.docker.yml
The original `.kitchen.yml` (for Vagrant driver) is kept clean:
https://github.com/chef-cookbooks/database/blob/master/.kitchen.yml